Source/JavaScriptCore/runtime/VM.cpp

```diff
@@ -174,11 +175,9 @@ static bool enableAssembler()
     if (!Options::useJIT())
         return false;
 
-    char* canUseJITString = getenv("JavaScriptCoreUseJIT");
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-    if (canUseJITString && !atoi(canUseJITString))
+    auto canUseJITString = span(getenv("JavaScriptCoreUseJIT"));
```

Comment:
I presume this is the `unsafeSpan`, but maybe this patch is going in before the rename.

Reply:
Yes, I'm still trying to get the `unsafeSpan()` PR to build on glib ports :'( This may land first.